types/index.d.ts

```diff
@@ -53,7 +53,7 @@ declare namespace fastifyRequestContext {
   }
 
   export const requestContext: RequestContext
-
+  export const asyncLocalStorage: AsyncLocalStorage<any>
```

Comment:
Not a big fan of forcing `any`. Maybe we can expose a potentially augmentable type within a module?